### PR TITLE
first steps toward getting started

### PR DESCRIPTION
**Pull request by matt-bernhardt**
This adds a "Getting Started" tutorial article. Right now it is at best half-written, with an emphasis on commands to run to set up the prerequisite tools.

The project readme will probably need to change as a result of writing this article, but I haven't thought about that yet.

### Secrets

- [ ] No secrets are affected

### Documentation

- [ ] Project documentation has been updated

### Accessibility

- [ ] ANDI or Wave has been run in accordance to
      [our guide](https://mitlibraries.github.io/guides/basics/a11y.html) and
      all issues introduced by these changes have been resolved or opened as new
      issues (link to those issues in the Pull Request details above)

### Stakeholder approval

- [ ] Stakeholder approval is not needed

### Dependencies

NO dependencies are updated

## Code Reviewer

- [ ] The commit message is clear and follows our guidelines
      (not just this pull request message)
- [ ] The changes have been verified
- [ ] The documentation has been updated or is unnecessary
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

File: docs/tutorial/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+### Composer
+
+https://getcomposer.org/
+
+```bash
+$ brew install composer
+```
+
+The only global package currently needed is PHP CodeSniffer, and the WordPress
```
Comment (JPrevost) [inline docs/tutorial/getting-started.md]: I never did any of these global phpcs steps. Maybe we add that to an advanced setup doc and focus this on the core seeing as how I never did this and didn't notice for this long lol?

Comment (JPrevost) [reply docs/tutorial/getting-started.md]: I'm 47% sure my `composer security` still works as expected but I may need help confirming.

Comment (matt-bernhardt) [reply docs/tutorial/getting-started.md]: Hmm... let me identify a step to confirm whether `composer security` does something. What happens now when you type that command?

Comment (matt-bernhardt) [reply docs/tutorial/getting-started.md]: FWIW, this is what I see when I run the command at the moment:

```bash
% composer security
> phpcs --standard=phpcs.security.xml
```

Comment (matt-bernhardt) [reply docs/tutorial/getting-started.md]: So, one way to confirm that Composer is successfully running the check:

1. Find any instance of `phpcs:disable` in the codebase, and remove that line. One example is this: https://github.com/MITLibraries/mitlib-wp-network/blob/master/web/app/themes/mitlib-child/functions.php#L214
2. With that line removed, there is now something for the security check to find. This should now be the output when running `composer security`

```bash
% composer security
> phpcs --standard=phpcs.security.xml

FILE: /Users/mjbernha/lando/mitlib-wp-network/web/app/themes/mitlib-child/functions.php
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 214 | ERROR | All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$output'.
     |       | (WordPress.Security.EscapeOutput.OutputNotEscaped)
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 1.39 secs; Memory: 10MB

Script phpcs --standard=phpcs.security.xml handling the security event returned with error code 1
```

Comment (matt-bernhardt) [reply docs/tutorial/getting-started.md]: In terms of this PR, if your setup works as described above with composer security, then I think I'm fine with moving this section to a "further setup" document or something? Does that sound good to you?

Comment (JPrevost) [reply docs/tutorial/getting-started.md]: Yeah that works the same for me after removing the two lines from the child themes functions.php.

```
[I] ⋊> ~/p/mitlib-wp-network on hours  composer security                                                                                                
> phpcs --standard=phpcs.security.xml

FILE: /Users/jprevost/projects/mitlib-wp-network/web/app/themes/mitlib-child/functions.php
----------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------------------------------------------------
 214 | ERROR | All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$output'.
     |       | (WordPress.Security.EscapeOutput.OutputNotEscaped)
----------------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 2.35 secs; Memory: 10MB
```

**Comment by matt-bernhardt** [comment]
Okay @JPrevost - I've split that section out to a separate tutorial, leaving behind a more minimal Composer setup step. Does this look better / more like what you followed?

File: docs/tutorial/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Please ensure you have the following tools set up on your computer. Everything
+can be installed via [Homebrew](https://brew.sh/).
+
+### PHP
+
+We currently use the latest release on the 8.0.x branch, using commands such as:
+
+```bash
+$ brew install php@8.0
```
Comment (JPrevost) [inline docs/tutorial/getting-started.md:15]:
```
php --version                                                                                                    15:31:19
PHP 8.2.4 (cli) (built: Mar 16 2023 16:25:32) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.4, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.4, Copyright (c), by Zend Technologies
```
uh... oops

**Review by JPrevost — APPROVED** [review]
I think this looks good. I'm sure we'll adjust as Adam goes through the process but having a place to point at and say "this part is confusing" will be great. Thanks!

File: docs/tutorial/getting-started.md (hunk 2 above)
Comment (matt-bernhardt) [reply docs/tutorial/getting-started.md:15]: Hopefully that bodes well for when we get around to upgrading PHP versions :-)

**Pull request merged**